dev/system-test/firestore.ts

```diff
@@ -4084,6 +4084,156 @@ describe('Query class', () => {
         docChanges: expectedChanges,
       });
     });
+
+    describe('sort unicode strings', () => {
+      it('snapshot listener sorts unicode strings same as server', async () => {
+        const collection = await testCollectionWithDocs({
+          a: {value: 'Łukasiewicz'},
+          b: {value: 'Sierpiński'},
+          c: {value: '岩澤'},
+          d: {value: '🄟'},
+          e: {value: 'Ｐ'},
+          f: {value: '︒'},
+          g: {value: '🐵'},
+        });
+
+        const query = collection.orderBy('value');
+        const expectedDocs = ['b', 'a', 'c', 'f', 'e', 'd', 'g'];
+
+        const getSnapshot = await query.get();
+        expect(getSnapshot.docs.map(d => d.id)).to.deep.equal(expectedDocs);
+
+        const unsubscribe = query.onSnapshot(snapshot =>
+          currentDeferred.resolve(snapshot)
+        );
+        const watchSnapshot = await waitForSnapshot();
+        snapshotsEqual(watchSnapshot, {
+          docs: getSnapshot.docs,
+          docChanges: getSnapshot.docChanges(),
+        });
+        unsubscribe();
```

Comment:
there are no `expect` statements after line 4104 in this test.  are you missing the `expect` statement for `watchSnapshot`? same for the other tests.

Reply:
`snapshotsEqual` asserts `watchSnapshot` has the same docs as getSnapshot.

Reply:
oops. not sure how I missed that :)

Reply:
LGTM